In [1]:
%matplotlib notebook
import pandas as pd
import numpy as np
import pickle
import matplotlib.pyplot as plt
import cira72 as ci

### Run test 

In [34]:
sun_d = 0
sun_ra = 0
sat_d = np.pi/6
sat_ra = -np.pi/4
f10 = 100
f10_mean = 91.4
kp = 0
height = 300
AMJD = 0

In [35]:
Temp,alion,AMMW,rho = ci.aden(AMJD,sun_ra,sat_d,sat_ra,sat_d,height,f10,f10_mean,kp)

In [36]:
Temp

[800.06254814222518, 783.71298417566004]

In [37]:
rho

1.4806298377457777e-12

### Table values of rho is 1.25E-11 and temp is 783.7

In [38]:
height = 400
Temp,alion,AMMW,rho = ci.aden(AMJD,sun_ra,sat_d,sat_ra,sat_d,height,f10,f10_mean,kp)

In [43]:
Temp

[800.07218841802592, 795.64616137693258]

In [39]:
rho

1.6688511382243043e-13

### Table values of rho is 1.34E-12 and temp 795.6

In [41]:
AMMW

15.238840063712573

In [42]:
alion

array([ 10.88773742,   9.27263177,  12.78269184,   6.17658121,
        11.43864361,  11.2523125 ])

## numerical example 15

In [52]:
sat_d = 45/180*np.pi
sat_ra = -120/180*np.pi

sun_d =-20/180*np.pi
sun_ra = -120/180*np.pi-np.pi/12
f10 = 136
f10_mean = 155
kp = 2
height = 300
AMJD = 19056  

In [53]:
Temp,alion,AMMW,rho = ci.aden(AMJD,sun_ra,sat_d,sat_ra,sat_d,height,f10,f10_mean,kp)

In [54]:
Temp

[1137.6807851331503, 1101.3555603928869]

### Temp in book is 1048 ; 1020

In [57]:
np.log10(rho)

-11.481724113258378

### log(Rho) in the book is -11.014 ; -11.026